python/ray/tune/experimental/output.py

```diff
@@ -557,6 +583,36 @@ def _print_heartbeat(self, trials, *sys_args):
 
 
 class TuneTerminalReporter(TuneReporterBase):
+    def experiment_started(
+        self,
+        experiment_name: str,
+        experiment_path: str,
+        searcher_str: str,
+        scheduler_str: str,
+        total_num_samples: int,
+        tensorboard_path: Optional[str] = None,
+        **kwargs,
+    ):
+        if total_num_samples > sys.maxsize:
+            total_num_samples_str = "infinite"
+        else:
+            total_num_samples_str = str(total_num_samples)
+
+        print(
+            tabulate(
+                [
+                    ["Search algorithm", searcher_str],
+                    ["Scheduler", scheduler_str],
+                    ["Number of trials", total_num_samples_str],
+                ],
+                headers=["Configuration for experiment", experiment_name],
+                tablefmt=AIR_TABULATE_TABLEFMT,
+            )
+        )
+        self._experiment_start_paths(
```

Comment:
instead of doing this, you can also ``super().experiment_start_paths()``?
then you don't need a private ``_experiment_start_paths()``.
btw what do ``paths`` mean in this context?

Reply:
Nice! Updated the code